src/js/Content/Features/Store/App/FSteamSpy.js

```diff
@@ -44,16 +44,16 @@ export default class FSteamSpy extends Feature {
         let _value = value;
         const result = [];
 
-        const days = Math.trunc(_value / 1440);
-        if (days > 0) { result.push(`${days}d`); }
+        const days = Localization.str.spy.playtime_unit_day.replace("__days__", Math.trunc(_value / 1440));
+        if (days > 0) { result.push(`${days}`); }
```

Comment:
Lastest commit broke things 😅
If you're gonna reformat, do it like this instead, same with hours and minutes:
```suggestion
        const days = Math.trunc(_value / 1440);
        if (days > 0) { result.push(`${Localization.str.spy.playtime_unit_day.replace("__days__", days)}`); }
```